src/routes/homepage/sections/hero.svelte

```diff
@@ -57,5 +57,14 @@
     }
     .background-image {
         z-index: -1;
+        width: 100%;
```

Comment:
this bg can be improved, for example, for big screens where the container is `1200px`, the background is more like

```css

    width: calc((100vw - 1200px) / 2 + 1200px + 10px);
    right: 0px;
    top: 113px;
    height: auto;
```
adapting this to the other screens I think would make the CSS more accurate to the original design

Reply:
we can apply the same to the FAQs, the code is more similar to the code above, wdyt?